src/utils.ts

```diff
@@ -209,6 +222,20 @@ function aliasFn(
     return fn.apply(context, args)
 }
 
+/**
+ * traverse up the scope chain until browser element was reached
+ */
+function getBrowserObject (elem: WebdriverIO.Element | WebdriverIO.Browser): WebdriverIO.Browser {
+    const elemObject = elem as WebdriverIO.Element
+    return (elemObject as WebdriverIO.Element).parent ? getBrowserObject(elemObject.parent) : elem as WebdriverIO.Browser
+}
+
+async function getFirstElement (received: WdioElementMaybePromise): Promise<WebdriverIO.Element> {
```

Comment:
Added helper to not duplicate logic of get first element instance to get a browser instance